src/loaders/ObjectLoader.js

```diff
@@ -477,9 +477,13 @@ Object.assign( ObjectLoader.prototype, {
 
 			return loader.load( url, function () {
 
+				if ( onLoad !== undefined ) onLoad( );
```

Comment:
I'm afraid this change is incorrect. The code will fire the global `onLoad()` function as soon as a single image is loaded. If we only use a single instance of `LoadingManager`, it's necessary to rewrite `parseImages()`.

Reply:
I suggest to close the PR and wait on @mrdoob's opinion. We should continue the discussion in #13177 and open a new PR if it's clear what we're going do...

Reply:
Good catch, of course the loading manager isn't gating the callback anymore. I grabbed a model with multiple textures for testing and pushed the fix. Obviously it's up to you guys what PRs you accept, but IMO this is a fix for a non-trivial oversight in the core model loader.